docs/maintenance/Pull_Requests.mdx

```diff
@@ -24,15 +24,85 @@ Per the scales from [Contribution Tiers > Pointing](./Contributor_Tiers.mdx#poin
+- Call out any positive aspects you see - especially in larger pull requests.
+- Use happy emojis frequently.
+- If you have the energy, post a fun (but not inappropriate) GIF with your review.
+  - We often use the _GIFs for Github_ extension: available as [GIFs for GitHub (Chrome)](https://chrome.google.com/webstore/detail/gifs-for-github/dkgjnpbipbdaoaadbdhpiokaemhlphep) and [GIFs for GitHub (Firefox)](https://addons.mozilla.org/en-US/firefox/addon/gifs-for-github).
+
+Pull requests are the first time many community members meaningfully interact with us - or, oftentimes, any open source maintainers at all.
+It's important that we provide a positive, uplifting experience. ❤️
+
+### Reviewing a PR
+
+Before reviewing a pull request, try to look back over the backing issue to (re-)familiarize yourself with it.
+You may find it useful to:
+
+- Attempt to simplify the provided reduction ("code golf")
+- Look back through previous issues and PRs around the same area of code / lint rule
+
+#### Posting Comments
+
+Post about one comment per area of praise note, suggested change, or non-actionable note.
+It's fine to use multiple ancillary comments to indicate _"(also here)"_ notes, but don't overwhelm with requests.
+
+:::tip
+If you're posting more than a couple types of comments, consider prefixing them with category indicators such as `[Docs]`, `[Praise]`, `[Testing]`, etc.
+This helps avoid the review feeling like a huge slog of big requests.
+:::
+
+Be clear in each of your comments what you're looking for or saying.
+Err on the side of providing more information than you think is needed.
+
+Try to default to a questioning tone for points that aren't clear bugs.
+Encourage authors to think on your suggestions: _"I think {xyz}, but am not sure - what do you think?"_.
+
+#### Posting Preliminary or Repeat Reviews
+
+Sometimes you may want to post a preliminary review, and/or realize later on you missed something in an earlier review.
+Both are reasonable and fine.
+
+For preliminary reviews, be clear with what scale you're reviewing at: _"Reviewing on architecture now, will look in more detail once it's settled"_.
+
+For repeat reviews, be clear when it's something you missed earlier and/or there's new information.
+Don't apologize if the missed information was only made clear because of changes due to comments - this is part of the review process!
+
+### Common Things to Look For
+
+- Style: that can you read through the code easily, there are comments for necessary tricky areas, and not unnecessary comments otherwise.
+  - Try not to nitpick things that don't matter.
+  - If something is standard in the repo but not enforced, consider mentioning it as a non-blocking comment and filing an issue to enforce it.
+- Thoroughness: does it handle relevant edge cases? Commonly:
+  - Generics and type parameters (see: `getConstrainedTypeAtLocation`).
+  - Parenthesis and whitespace (see: `getWrappingFixer`).
```

Comment:
```suggestion
  - Parenthesis and whitespace (see: `getWrappingFixer`).
  - Unions and intersections (see: `unionTypeParts` and `intersectionTypeParts`).
```

IMO, this is a pretty common edge case, isn't it? :smile:

Reply:
Yeah! Good point. +1